sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server-obo/README.md

```diff
@@ -48,68 +55,44 @@ We need to ensure that this [environment checklist][ready-to-run-checklist] is c
 </dependencies>
 ```
 
-### Register your Web API
-You can follow *azure-spring-boot-sample-active-directory-resource-server* sample to add `ResourceAccessGraph.read`, `ResourceAccessGraphCustomResources.read` scopes. 
-Convention current application id url is `api://sample-client-id`, ; the application id url of the sample *azure-spring-boot-sample-active-directory-resource-server* is `custom-client-id`.  
-After adding as shown below:
-
-   ![API Permissions](resource/resource-server-obo-add-scope.png)
-
-### Add API permissions
-The current Web API will access Graph API and Custom API. 
-
-Sign in to the [Azure portal][azure-portal]. If you have access to multiple tenants, use the **Directory + subscription** filter  in the top menu to select the tenant containing your client app's registration.
-
-#### Add Graph API Permission
-1. Select **Azure Active Directory** > **App registrations**, and then select your current sample application (not your web API).
-
-2. Select **API permissions** > **Add a permission** > **Microsoft APIs** > **Microsoft Graph** > **Delegated permissions**, select **User.Read**, select **Add permission** to complete the process.
+### Configure your middle-tier Web API A
+1. You can follow *Web API A* sample to add `ResourceAccessGraph.Read`, `ResourceAccessGraphCustomResources.Read` scopes. Convention current application id url is `api://xxxxx-xxxx-xxxxxx`.(**NOTE**: In the **Web API A** `ResourceAccessGraph.Read`, `ResourceAccessGraphCustomResources.Read` scopes should be added to the **Web APP**.)![API Permissions](docs/image-resource-server-obo-add-scope.png)
```

Comment:
Did we mention the Web API sample link somewhere above?

Reply:
Maybe `Convention of current application id` is better.

Reply:
Why do we have a concept of `Web APP` here?

Reply:
`Did we mention the Web API sample link somewhere above?`
Yes. A link to a sample Web API already exists above.

`Maybe Convention of current application id is better.`
I agree with you.

`Why do we have a concept of Web APP here?`
I want to tell users of this sample that they should configure their Web API A permissions in their web app.